docs/api-guide.md

```diff
@@ -295,6 +295,25 @@ boolean ifExist = true;
 admin.dropCoordinatorTables(ifExist);
 ```
 
+### Import a table
+
+You can import an existing table to ScalarDB as follows:
+
+```java
+// Import the table "ns.tbl". If the table is already managed by ScalarDB, the target table does not
+// exist, or the table does not meet the requirement of ScalarDB table, an exception will be thrown.
+admin.importTable("ns", "tbl");
```

Comment:
We have added `Map<String, String> options` as 3rd argument to this method:
```suggestion
admin.importTable("ns", "tbl", options);
```

And we didn't add the argument for the `3` branch. To avoid diverging this doc between `master` and `3`, so we should probably apply the same change for the `3` branch.

Reply:
Good catch, thank you! Fixed in e3dd00d.

> And we didn't add the argument for the 3 branch. To avoid diverging this doc between master and 3, so we should probably apply the same change for the 3 branch.

I'm OK to use `admin.importTable("ns", "tbl", options)` even in v3.x, but might be confusing for users. If we can just apply it without any other concerns, I can handle it since it's not a big diverge. Should I or not?

Reply:
Actually, the `importTable()` method was already introduced in 3.10, but it was treated as an experiment feature. Therefore, I think we can also add the `options` argument to the `3` branch.

Reply:
Ahh, right. Got it. Thank you!